packages/react/src/table/TableScroller.tsx

```diff
@@ -341,6 +363,41 @@ export function TableScroller({ children }: TableScrollerProps) {
 	);
 }
 
+function Shadow({
+	edge,
+	height,
+	isVisible,
+}: {
+	edge: 'left' | 'right';
+	height: number;
+	isVisible: boolean;
+}) {
+	return (
+		<Box
+			css={{
+				height,
+				opacity: isVisible ? 1 : 0,
+				pointerEvents: 'none',
+				position: 'absolute',
+				top: 0,
+				transition: `opacity ${tokens.transition.duration}ms ${tokens.transition.timingFunction}`,
+				width: 28,
+				...(edge === 'left'
+					? {
+							background:
+								'linear-gradient(to right, rgba(0, 0, 0, 0.08), transparent)',
```

Comment:
Is it possible to use the token instead of the `0`s for RGB so that we cover off dark mode?

Reply:
Shadow tokens currently don't respond to dark mode (they're hardcoded as the same), so this was just a 1:1 mapping of how our shadow tokens render without actually being able to use them

Reply:
I was thinking along the lines of `foregroundText` but this sounds like another ticket.